clients/feeder/feeder.go

```diff
@@ -61,6 +61,11 @@ func (c *Client) WithUserAgent(ua string) *Client {
 	return c
 }
 
+func (c *Client) WithTimeout(t time.Duration) *Client {
+	c.client.Timeout = t
```

Comment:
This timeout includes the time spent reading the body. Is that something we want? ([reference](https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/)) Maybe `IdleConnTimeout` would be better?

Reply:
Hmm, I dont see any other configurable timeouts in `http.Client`? Do we need to override its `Transport` ?

Reply:
Yes. `net.Dialer.Timeout` (TCP connection setup timeout) and `http.Client.Timeout` are the only client timeouts not set on the `Transport`.

Reply:
Let's not rush doing that, 5secs is plenty of time to download a block and state update.